PublicGitArchive/web/md/index.md

```diff
@@ -0,0 +1,50 @@
+# 190k repositories dataset
+
+On this dataset you can find all the repositories from GitHub with more than 50 stars.
+
+## Links
+
+- ### [CSV file](LINK TO CSV FILE HERE)
+- ### [SIVA files](LINK TO SIVA FILES HERE)
+
+## Files explanation
+
+Description of the content of each file type and explanation about some key concepts.
+
+You can download and filter by csv data using [multitool](https://github.com/src-d/datasets/tree/master/PublicGitArchive).
+
+
+### CSV with metadata per repository
+A CSV with useful data per each repository on the dataset. The columns on the CSV are:
+
+|  Column name | Column description  |
+|---|---|
+|URL|The url of the GitHub repository that allow us to access to the Github web interface.|
```

Comment:
Needs to be synced with https://github.com/src-d/papers/blob/master/internal/data_showcase/data_showcase.tex#L212 - no worries, will do myself as I had to change the descriptions to squeeze the paper into 4 pages. Also here is upper case - will change to lower case - again, otherwise the paper layout explodes because CAPS are wider.